gensim/sklearn_api/w2vmodel.py

```diff
@@ -83,4 +83,5 @@ def transform(self, words):
         return np.reshape(np.array(X), (len(words), self.size))
 
     def partial_fit(self, X):
-        raise NotImplementedError("'partial_fit' has not been implemented for W2VTransformer")
+        raise NotImplementedError("'partial_fit' has not been implemented for W2VTransformer since 'update()' function for Word2Vec class is experimental. "
```

Comment:
Ah, I see. So the issue is actually "updating the vocab" (and not "training") being experimental. I have updated the error message in `partial_fit` for `W2VTransformer` for this accordingly.
Thanks for your explanation and suggestions about resolving this concern. I guess it would be nice to create an issue for this. :)